### PR TITLE
Basic CRUD for jobs

### PR DESCRIPTION
**Pull request by fishnix**

### CODE REVIEW

**Comment by damnski** [comment]
Very nice..

File: api/handlers_jobs.go [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+	w = LogWriter{w}
+	vars := mux.Vars(r)
+	account := vars["account"]
+	if _, ok := s.accounts[account]; !ok {
+		msg := fmt.Sprintf("account not found: %s", account)
+		handleError(w, apierror.New(apierror.ErrNotFound, msg, nil))
+		return
+	}
+
+	log.Debugf("creating job for account %s", account)
```
Comment (tenyo) [inline api/handlers_jobs.go]: Info?

File: api/handlers_jobs.go [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+	account := vars["account"]
+	id := vars["id"]
+
+	if _, ok := s.accounts[account]; !ok {
+		msg := fmt.Sprintf("account not found: %s", account)
+		handleError(w, apierror.New(apierror.ErrNotFound, msg, nil))
+		return
+	}
+
+	log.Debugf("deleting job '%s' for account %s fromt repository", id, account)
```
Comment (tenyo) [inline api/handlers_jobs.go]: Info?

File: jobs/job.go [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+				msg := fmt.Sprintf("schedule_expression is not a valid cron expression: '%s': %s", s, err)
+				return errors.New(msg)
+			}
+			m.ScheduleExpression = s
+		}
+	}
+
+	if enabled, ok := rawStrings["enabled"]; ok {
+		if s, ok := enabled.(bool); !ok {
+			msg := fmt.Sprintf("enabled is not a string: %+v", rawStrings["enabled"])
```
Comment (tenyo) [inline jobs/job.go]: 👀 `bool`

File: jobs/s3repository.go [hunk 4]
```diff
@@ -0,0 +1,291 @@
...
+
+	// generate a new random ID for the job
+	job.ID = NewID()
+
+	return s.Update(ctx, account, job.ID, job)
+}
+
+// Delete deletes a job in the s3 jobs repository
+func (s *S3Repository) Delete(ctx context.Context, account, id string) error {
+	if id == "" {
```
Comment (tenyo) [inline jobs/s3repository.go]: Should also check if the `account` is empty

File: jobs/s3repository_test.go [hunk 5]
```diff
@@ -0,0 +1,484 @@
...
+			"6":  "whiplash",
+			"7":  "phantom lord",
+			"8":  "no remorse",
+			"9":  "seek & destroy",
+			"10": "metal militia",
+		},
+		Enabled:            true,
+		ModifiedAt:         &testTime,
+		ModifiedBy:         "hetfield",
+		Name:               "kill 'em all",
```
Comment (tenyo) [inline jobs/s3repository_test.go]: 🤘

**Pull request merged**